riak_repl/datadog_checks/riak_repl/riak_repl.py

```diff
@@ -10,50 +9,68 @@
 
 class RiakReplCheck(AgentCheck):
 
-    REPL_STATS = [
-        "server_bytes_sent",
-        "server_bytes_recv",
-        "server_connects",
-        "server_connect_errors",
-        "server_fullsyncs",
-        "client_bytes_sent",
-        "client_bytes_recv",
-        "client_connects",
-        "client_connect_errors",
-        "client_redirect",
-        "objects_dropped_no_clients",
-        "objects_dropped_no_leader",
-        "objects_sent",
-        "objects_forwarded",
-        "elections_elected",
-        "elections_leader_changed",
-        "rt_source_errors",
-        "rt_sink_errors",
-        "rt_dirty",
-        "realtime_send_kbps",
-        "realtime_recv_kbps",
-        "fullsync_send_kbps",
-        "fullsync_recv_kbps",
-    ]
-
-    REALTIME_QUEUE_STATS = ["percent_bytes_used", "bytes", "max_bytes", "overload_drops"]
-
-    FULLSYNC_COORDINATOR = [
-        "queued",
-        "in_progress",
-        "waiting_for_retry",
-        "starting",
-        "successful_exits",
-        "error_exits",
-        "retry_exits",
-        "soft_retry_exits",
-        "busy_nodes",
-        "fullsyncs_completed",
-        "last_fullsync_duration",
-    ]
+    REPL_STATS = {
+        "server_bytes_sent": "gauge",
```

Comment:
I had been going back and forth on submitting some of the metrics as counts because that is what they are designated as in riak_repl. However, I settled on leaving them as gauges because that seemed simpler and more flexible once the data reached datadog. I left the work incase I changed my mind in the future.